src/main/java/teammates/common/datatransfer/FeedbackResponseCommentAttributes.java

```diff
@@ -191,9 +191,7 @@ public String getJsonString() {
     @Override
     public void sanitizeForSaving() {
         this.commentText = Sanitizer.sanitizeTextField(this.commentText);
```

Comment:
So `sanitizeForRichText` will do something like this

`return new Text(Sanitizer.sanitizeForRichText(sanitizeTextField(text.getValue())));`